esp8266/main.c

```diff
@@ -108,6 +108,7 @@ void init_done(void) {
     #endif
 }
 
+// Seems unused
```

Comment:
Unrelated change.

Also, "seems" sounds dangerous. This patch also "seems" to work, right?